base_changeset/models/base.py

```diff
@@ -114,7 +114,7 @@ def write(self, values):
         for record in self:
             local_values = self.env["record.changeset"].add_changeset(record, values)
             super(Base, record).write(local_values)
-        return self
+        return bool(self)
```

Comment:
I would say, simply return True. But can you propose against oca/12.0 so that we can have that discussion with the wider community? To fix the ticket (and to saveguard against any upstream bugs or changes in the future) it is enough to have https://github.com/VanMoof/odoo-addons/pull/892

Reply:
I would vote for return True and propose against oca/12.0

Reply:
@StefanRijnhart do you have any other comments?

Reply:
My comment would be that we don't need this PR. There is the upstream PR to fix the issue for the rest of the world, and there is https://github.com/VanMoof/odoo-addons/pull/892 to fix the return of the store.api which is a good idea in case any other module starts breaking this all of a sudden.